### PR TITLE
Logic Update for Mengubah Step By Step

### PR DESCRIPTION
**Pull request by swastikanata**
When updating status for steps, update status for task as well

### CODE REVIEW

**Review by carlenee — DISMISSED** [review]
can you add the get api for the task steps that was deleted or maybe not if it's not necessary anymore? whatever suits our project more thank you

File: task_steps/views.py
Comment (JohannesSetiawan) [inline task_steps/views.py:1]: I guess for task instance, the status is "Not Started", "On Progress", and "Done" not NOT_STARTED, ON_PROGRESS, DONE. Might want to check wether Line 108 - 113 really works as expected.

**Review by JohannesSetiawan — CHANGES_REQUESTED** [review]
Overall it looks good. But might want to do some checking first

**Review by JohannesSetiawan — APPROVED** [review]

**Review by catherineeangel — APPROVED** [review]
LGTM

**Pull request merged**